### ISSUE TITLE
[Question] Change Keybind

### ISSUE THREAD

**Issue by fgsfdss** [shirsig/Mail#37]
Right-clicking on items in your inventory to mail them is convenient, but can be risky if you're absent-minded (I'm thinking consumables).
Is it possible to change the right-click functionality to alt-click instead?

**Comment by shirsig** [comment]
Until some time ago this functionality was disabled in the inbox tab which I think someone complained about and I agreed was dangerous, so I made right clicking when in the inbox tab cause switching to the send tab and also attach the item. After that change I don't think there's any notable danger anymore. Doesn't seem realistic that you accidentally right click an item while not even at the mailbox in the hope to attach it.

**Comment by fgsfdss** [comment]
To come back to this, when alt-tabbing out on the loading screen and back in again after loading completed, the alt input gets stuck. I attempted to mail some pots from my bank alt to my main to be met with a "you can't use that item" right clicking them, of course as the game took it as an alt-right click. If I had mailed them the other way around they would in fact have been wasted.

**Comment by shirsig** [comment]
cannot reproduce this. In any case, that would be an issue with tons of addons using unmodified clicks and or alt-clicks. To do defensive coding around people getting their keys stuck is ridiculous.

**Comment by fgsfdss** [comment]
Well yes, it is, however nowhere really significant as the worst that could happen is pressing a wrong keybind. Sending something like a flask over and accidentally using it would be a lot more problematic.
Might be a problem with my client. Either way, this was not a feature request to begin with, but rather a question on how to potentially edit it myself. I had a look but my lua knowledge is too limited to find and likely also to edit the event.

**Comment by fgsfdss** [comment]
I don't use Lazypig so that's alright. If I just remove "or IsAltKeyDown()", would that cause any issues elsewhere?

**Comment by shirsig** [comment]
No, but it'll still work without alt too. It was just the first example that came to mind, there are countless addons that will fuck you to varying degrees if your keyboard doesn't work properly, after all addons can do all kinds of dangrous things like destroying items/gold, buying auctions, flag you for pvp etc.

**Comment by fgsfdss** [comment]
I wish I could grab the problem by its root but as it happens on my laptop as well, I assume the only option would be to find and install a different client. Not sure yet if it's worth that amount of trouble, as due to my keyboard layout I usually notice the stuck alt input quickly by moving forward with Z (it'll hide my UI instead of moving forward). I only just encountered this issue as my bank alts are stationed right next to the mailbox which does not require movement, and I used CT_Mailmod which uses alt-click until recently when I switched to Mail.
Don't worry, I've not yet insta-bought anything with Aux.
Will try this bandage-fix out for the time being either way. Thanks for the help.
